src/lib/zapUtils.ts

```
@@ -0,0 +1,78 @@
+  }
+
+  return null;
+}
+
+
+
+export function makeZapRequest({ profile, event, amount, relays = [], comment = '' }) {
+  if (!amount) throw new Error('Amount not given');
+  if (!profile) throw new Error('Profile not given');
+
+  let zapRequest = {
```

Biome

[error] This let declares a variable that is only assigned once.

'zapRequest' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)